RoadMap_SwiftUI/RoadMap_SwiftUI/View/SettingsView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  SettingsView.swift
+//  RoadMap_SwiftUI
+//
+//  Created by Артур Щукин on 26.01.2023.
+//
```

Comment:
```suggestion
//
//  SettingsView.swift
//  RoadMap_SwiftUI
//
//  Created by Артур Щукин on 26.01.2023.
//
```
не отработал форматер